lib/BuiltinsGen.cpp

```diff
@@ -0,0 +1,1083 @@
+
+    // Async
```

Comment:
You can protect the formatting of this with comments like:

     // clang-format off

...

      // clang-format on